kafka/channel/pkg/controller/channel/provider_test.go

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
should we update all these dates to 2019?

Reply:
ok, can do that in a different PR